modules/RTC/TPCUtils.js

```diff
@@ -579,25 +606,26 @@ export class TPCUtils {
         }
         const rids = [
             {
-                id: SIM_LAYER_1_RID,
+                id: SIM_LAYERS[0].rid,
                 direction: 'recv'
             },
             {
-                id: SIM_LAYER_2_RID,
+                id: SIM_LAYERS[1].rid,
                 direction: 'recv'
             },
             {
-                id: SIM_LAYER_3_RID,
+                id: SIM_LAYERS[2].rid,
                 direction: 'recv'
             }
         ];
 
         // Firefox 72 has stopped parsing the legacy rid= parameters in simulcast attributes.
         // eslint-disable-next-line max-len
         // https://www.fxsitecompat.dev/en-CA/docs/2019/pt-and-rid-in-webrtc-simulcast-attributes-are-no-longer-supported/
+        const ridLine = rids.map(val => val.id).join(';');
         const simulcastLine = browser.isFirefox() && browser.isVersionGreaterThan(71)
```

Comment:
While we're here, do we support FF < 71?

Reply:
Looks like we don't have a min. version check for FF in the library. I think we should do that in a separate PR.